activiti-core/activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/activiti/runtime/api/impl/ProcessRuntimeImpl.java

```diff
@@ -555,4 +550,9 @@ private boolean isATaskAssigneeOrACandidate(String processInstanceId) {
         return taskQuery.count() > 0;
     }
 
+    private List<String> getCurrentUserGroupsIncludingEveryOneGroup() {
+        List<String> groups = new ArrayList<>(securityManager.getAuthenticatedUserGroups());
+        groups.add("*");
```

Comment:
is "*" a group? Can we make it more explicit? use another name or set a constant with a meaningful name?

Reply:
I'm trying to avoid a specific group name since its not a real group that exists.

Reply:
Regarding tests `nonCandidateStarter_should_getProcessDefinitionWithNoCandidatesSet` covers that. Additionally, by default now no one can get a process definition or start a process if the user is either a candidate or the process definition has an identitylink to the `*` group